cypress/support/network_commands.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Read indices from a file and send to OpenSearch using the create index API
+ * @param {String} filename File path (with its root at the directory containing the cypress.json file)
+ * @param {String} hostname Host name for OpenSearch
+ * @param {String} port Port for OpenSearch
+ */
+
+Cypress.Commands.add('importJSONMapping', (filepath, hostname = 'localhost', port = '9200') => {
+  cy.readFile(filepath, 'utf8').then((str) => {
+    const strSplit = str.split('\n\n')
+    cy.wrap(strSplit).each((element) => {
+      const json = JSON.parse(element)
+      if (json.type === 'index') {
+        const index = json.value.index
+        const settings = json.value.settings
+        const mappings = json.value.mappings
+        const aliases = json.value.aliases
+        const body = { settings, mappings, aliases }
+        cy.request({ method: 'PUT', url: hostname + ':' + port + '/' + index, body: body, failOnStatusCode: false }).then((response) => {
```

Comment:
format and pretty code/json

Reply:
Would you be able to clarify in what way the format should be changed? Is it changing the use of the constant variables and/or the format of the `cy.request` call?